monitoring/oplog-populator/dashboard.py

```diff
@@ -0,0 +1,335 @@
+        expr='\n'.join([
+            'sum(up{',
+            '    namespace="${namespace}",',
+            '    job="${oplog_populator_job}"',
+            '})',
+        ]),
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+kafka_connect_up_current = Stat(
+    title='Kafka Connect Instances',
+    description='Number of active kafka-connect instances',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='background',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='\n'.join([
+            'sum(up{',
+            '    namespace="${namespace}",',
+            '    job="${kafka_connect_job}"',
+            '})',
+        ]),
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+connector_instances = Stat(
+    title='Active Connectors',
+    description='Number of active kafka-connect connector instances',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='sum(' + Metrics.CONNECTORS.raw() + ')',
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+reused_connector_instances = Stat(
+    title='Reused Connectors',
+    description='Number of active kafka-connect connectors not created by the current oplog populator instance',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='sum(' + Metrics.CONNECTORS.raw(isOld='true') + ')',
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+total_number_buckets = Stat(
+    title='Total Buckets',
+    description='Number of buckets configured on the connectors',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='sum(' + Metrics.BUCKETS.raw() + ')',
+    )],
+    thresholds=[
+        Threshold('green', 0, 0.),
+    ])
+
+bucket_distribution = PieChart(
+    title='Bucket Distribution',
+    description='How buckets are distributed among the connectors',
+    dataSource='${DS_PROMETHEUS}',
+    legendValues=['percent', 'value'],
+    pieType='donut',
+    targets=[Target(
+        expr='sum by(connector) (' + Metrics.BUCKETS.raw() + ')',
+        legendFormat='{{connector}}',
+    )])
+
+max_pipeline_size = Stat(
+    title='Max Pipeline Size Per Connector',
+    description='Maximum size of the mongo pipeline per connector',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
+    reduceCalc='last',
+    format=UNITS.BYTES,
+    orientation='horizontal',
+    graphMode='none',
+    targets=[Target(
+        expr='sum by(connector) (' + Metrics.PIPELINE_SIZE.raw() + ')',
```

Comment:
i don't think we need to display the per-connector graph: just the overall value is enough (and specific connector can be filtered if needed)